packages/moleculer-db/src/index.js

```diff
@@ -105,6 +109,10 @@ module.exports = {
 					{ type: "string", optional: true },
 					{ type: "array", optional: true, items: "string" },
 				],
+				excludeFields: [
+					{ type: "string", optional: true },
```

Comment:
Add the string -> array conversion to `sanitizeParams` method, similar to `fields`